docs/_src/language/basic.md

```diff
@@ -1,54 +1,45 @@
-# Malloy Quick Start for SQL Programmers
+# Malloy Quickstart
 
-All Malloy compiles to SQL, so all Malloy gestures have a SQL translation. This doc is intended to serve as an introduction to the language, mapping Malloy syntax to SQL concepts.
+This document will quickly get you up to speed on what
+Malloy looks like, how to write queries, and how to
+save metrics and other analysis in _explores_. For detailed
+information on any of the topics in this document, see the
+following several sections.
 
-_Note: If you'd like to follow along with this guide, you can create a new, blank `.malloy` file and run these queries there. You can have one or multiple queries in a given `.malloy` file; the "Run" code lens option will let you select which query to run, otherwise cmd+enter will run the last query in the file._
+_Note: If you'd like to follow along with this guide, you can create a new <code>.malloy</code> file and run these queries there._
 
-Consider the simple Malloy query below.
+## Leading with the Source
+
+In Malloy, the source of a query is always first, and can be either a database table, a modeled explore (a table with relationships and calculations built-in), or even another query.
```

Comment:
So Lloyd had "physical table" before, but that sounded sort of nonsensical to me. Maybe that's an industry term I haven't heard before, but it sounds to me like you're talking about a "physical" hard drive—and especially when we're living in a context of BQ, that's a weirdly specific thing to draw to mind. I'm okay with "raw table", but I thought "database table" was clearest, because that's the point—it's a table that's in a database, rather than one that's virtualized in our software.

Reply:
I think physical table is a pretty broadly known term, to me "database table" sounds odd 🤷‍♀️  raw works for me

Reply:
we could also say "a table or view in the underlying database"